test/test_private.py

```diff
@@ -0,0 +1,21 @@
+from testing_base import *
+
+def test_twoCards():
```

Comment:
Would it be a good idea to add two players and test that the cards are only available to the correct player.  Is there currently any way to show the cards that are hidden?

Reply:
That is a good idea, I have made all the changes to ensure the cards are available to the correct player and that it displays the cards that are hidden. 
<img width="849" alt="Screenshot 2024-04-05 at 9 37 09 PM" src="https://github.com/TMUProfLee/CIS322_SP24/assets/69409957/dd739e55-41d0-4b19-b084-b4f0d08e6312">